api/v1alpha1/restore_types.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 The SODA Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// RestoreSpec defines the desired state of Restore
+type RestoreSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// BackupName is backup CR name specified during backup
+	BackupName string `json:"backupName,omitempty"`
+
+	// IncludeNamespaces are set of namespaces name considered for restore
+	IncludeNamespaces []string `json:"includeNamespace,omitempty"`
+
+	// ExcludeNamespaces are set of namespace name should not get considered for restore
+	ExcludeNamespaces []string `json:"excludeNamespaces,omitempty"`
+
+	// IncludeResources are set of kubernetes resource name considered for restore
+	IncludeResources []string `json:"includeResources,omitempty"`
+
+	// ExcludeResources are set of kubernetes resource name should not get considered for restore
+	ExcludeResources []string `json:"excludeResources,omitempty"`
+
+	// LabelSelector are label get evaluated against resource selection
+	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty"`
+
```

Comment:
RestorePVs (boolean) field to be added to specify whether need to restore PVs or not

Reply:
The field is expected with Volume support feature